src/design-system/color/palettes.ts

```diff
@@ -5,7 +5,7 @@ export const colors = {
   appleBlueLight: '#1F87FF',
   black: '#000000',
   blackTint: '#12131a',
-  grey: 'rgb(60, 66, 82)',
+  grey: '#3C4252',
```

Comment:
checking if this one was on purpose, we had a change like this in the last profiles PR

Reply:
was intentional, we use some transforms like `color + '10'` to render a color at 10% opacity, but that only works with hex colors. this was the only solid color in palettes.ts that still used rgb, so `grey + '10'` was breaking